Demo/src/main/java/com/braintreepayments/demo/CardFragment.java

```diff
@@ -443,6 +445,17 @@ private ThreeDSecureRequest threeDSecureRequest(PaymentMethodNonce paymentMethod
         threeDSecureRequest.setV2UiCustomization(v2UiCustomization);
         threeDSecureRequest.setV1UiCustomization(v1UiCustomization);
 
+        threeDSecureRequest.setUiType(ThreeDSecureRequest.BOTH);
+
+        ArrayList renderTypes = new ArrayList<>();
+        renderTypes.add(ThreeDSecureRequest.OTP);
+        renderTypes.add(ThreeDSecureRequest.SINGLE_SELECT);
+        renderTypes.add(ThreeDSecureRequest.MULTI_SELECT);
+        renderTypes.add(ThreeDSecureRequest.OOB);
+        renderTypes.add(ThreeDSecureRequest.RENDER_HTML);
+
+        threeDSecureRequest.setRenderType(renderTypes);
```

Comment:
Take it or leave it comment: might be more readable to use a literal:


```
        ArrayList renderTypes = new ArrayList<Integer>(Arrays.asList(
                ThreeDSecureRequest.OTP,
                ThreeDSecureRequest.SINGLE_SELECT,
                ThreeDSecureRequest.MULTI_SELECT,
                ThreeDSecureRequest.OOB,
                ThreeDSecureRequest.RENDER_HTML
        ));
        threeDSecureRequest.setRenderType(renderTypes);
```

Reply:
This also works. Pick your poison I guess

```
        ArrayList renderTypes = new ArrayList(){{
            add(ThreeDSecureRequest.OTP);
            add(ThreeDSecureRequest.SINGLE_SELECT);
            add(ThreeDSecureRequest.MULTI_SELECT);
            add(ThreeDSecureRequest.OOB);
            add(ThreeDSecureRequest.RENDER_HTML);
        }};
        threeDSecureRequest.setRenderType(renderTypes);
```

Reply:
```suggestion
        threeDSecureRequest.setRenderType(Arrays.asList(
                ThreeDSecureRequest.OTP,
                ThreeDSecureRequest.SINGLE_SELECT,
                ThreeDSecureRequest.MULTI_SELECT,
                ThreeDSecureRequest.OOB,
                ThreeDSecureRequest.RENDER_HTML
        ));
```
Yeah a good combination of those works. The biggest thing is declaring the property type as `List<Integer>` everywhere to keep Java happy.

Reply:
Done: https://github.com/braintree/braintree_android/pull/738/commits/b1313fc2f152519e3fea12c0de3b51e48e1364f7